netjsonconfig/backends/raspbian/templates/hostname.jinja2

```diff
@@ -0,0 +1,11 @@
+{% for i, j in data.items() %}
+{% if i|string() == 'general' %}
+{% for general in j %}
+{% if general.get('hostname') %}
+config: /etc/hostname
```

Comment:
are you sure this line doesn't make the configuration invalid when deployed on an actual device?

Shouldn't it be `# config: /etc/hostname`?